src/e2e/specs/breachResolution.spec.ts

```diff
@@ -110,45 +42,41 @@ test.describe("Breaches Dashboard - Headers", () => {
     await settingsPage.open();
 
     // verify logo and profile button
-    await expect(async () => {
-      expect(
-        await dataBreachPage.dataBreachesLogoLink.first().getAttribute("href"),
-      ).toBe("/user/breaches");
-    }).toPass();
+    expect(
+      await dataBreachPage.dataBreachesLogoLink.first().getAttribute("href"),
+    ).toBe("/user/breaches");
 
     await page.waitForLoadState("networkidle");
     await dataBreachPage.dataBreachesNavbarProfile.click();
 
-    // verify mnanage your ff account link, settings option, help and support option, sign out option
-    await expect(async () => {
-      // menu is open
-      expect(await dataBreachPage.profileMenuExpanded()).toBeTruthy();
-
-      // menu header
-      await expect(
-        dataBreachPage.dataBreachesNavbarProfileMenuHeader,
-      ).toBeVisible();
-
-      // head text
-      expect(
-        await dataBreachPage.dataBreachesNavbarProfileMenuHeaderSubtitle.textContent(),
-      ).toEqual("Manage your Mozilla account");
-
-      // check settings
-      await expect(
-        dataBreachPage.dataBreachesNavbarProfileMenuSettings,
-      ).toBeVisible();
-
-      // help and support
-      await expect(
-        dataBreachPage.dataBreachesNavbarProfileMenuHelpAndSupport,
-      ).toBeVisible();
-
-      // sign out
-      await expect(
-        dataBreachPage.dataBreachesNavbarProfileMenuSignOut,
-      ).toBeVisible();
-    }).toPass();
+    // verify manage your Mozilla account link, settings option, help and support option, sign out option
+    // menu is open
+    expect(await dataBreachPage.profileMenuExpanded()).toBeTruthy();
+
+    // menu header
+    await expect(
+      dataBreachPage.dataBreachesNavbarProfileMenuHeader,
+    ).toBeVisible();
+
+    // head text
+    expect(
+      await dataBreachPage.dataBreachesNavbarProfileMenuHeaderSubtitle.textContent(),
+    ).toEqual("Manage your Mozilla account");
```

Comment:
For atomic values, you should be able to use `.toBe`. `toEqual` is used to also check object properties, and in those cases, you'll usually want to use `toStrictEqual` anyway, which is why this version stands out a bit as a red flag to me.

```suggestion
    ).toBe("Manage your Mozilla account");
```

Reply:
Alternatively, you can even use the more specific matcher [`toHaveText()`](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text), which presumably gives more helpful error messages on test failures:

```ts
    expect(
      dataBreachPage.dataBreachesNavbarProfileMenuHeaderSubtitle
    ).toHaveText("Manage your Mozilla account");
```